packages/slice-machine/src/modules/selectedSlice/screenshot.ts

```diff
@@ -3,8 +3,6 @@ import { ScreenshotUI } from "@lib/models/common/ComponentUI";
 import { ScreenshotResponse, Screenshots } from "@models/common/Screenshots";
 
 export async function generateScreenShot(
```

Comment:
this function isn't used anymore now that we move the logic into sagas. Should i remove it?

Reply:
I'd say so it's dead code now, same should be for the generateCustomScreenshot.+ save + push yeah?

Reply:
can you also delete the save and push api calls that are not used anymore?